src/app/modules/shared_models/token_list_model.nim

```diff
@@ -76,6 +78,8 @@ QtObject:
       ModelRole.Image.int:"icon",
       ModelRole.Category.int:"category",
       ModelRole.CommunityId.int:"communityId",
+      ModelRole.Supply.int:"Supply",
+      ModelRole.IsInfiniteSupply.int:"IsInfiniteSupply",
```

Comment:
```suggestion
      ModelRole.IsInfiniteSupply.int:"isInfiniteSupply",
```

Reply:
Role names have to start with a lowercase letter, and we do already refer to them throughout the code as e.g. `model.supply`

Reply:
This should even read `infiniteSupply` as that's what all the UI elements use currently:

![image](https://github.com/status-im/status-desktop/assets/5377645/9c1174ad-b524-43c9-ba7e-0251c70f7e8b)